api/tests/routes/external/user_subscription_test.py

```diff
@@ -1910,6 +1910,73 @@ def test_decision_document_expired(self, client, ubble_mocker, ref_data_check_sc
 
         self._assert_email_sent(user, 384)
 
+    def test_decision_document_not_authentic(self, client, ubble_mocker):
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏On peut aussi tester le cas où retry = false